### PR TITLE
added sgETH test script

### PR DESCRIPTION
**Pull request by devlancer412**

### CODE REVIEW

File: hardhat.config.js [hunk 1]
```diff
@@ -108,10 +108,9 @@ module.exports = {
     // apiKey: secrets.ETHERSCAN_API,
     apiKey: {
       mainnet: process.env.ETHERSCAN_API,
-      goerli: process.env.ETHERSCAN_API
+      goerli: process.env.ETHERSCAN_API,
```
Comment (chimera-defi) [inline hardhat.config.js:111]: lets remove any files from the commit were not making many changes in. itll help me avoid merge conflicts later

File: test/v2/core/sgETH.spec.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    sgEth = await SgETH.deploy([]);
+    await sgEth.deployed();
+
+    deployer = owner;
+    alice = addr1;
+
+    MINTER_ROLE = await sgEth.MINTER();
+  });
+
+  it("addMinter", async () => {
```
Comment (chimera-defi) [inline test/v2/core/sgETH.spec.js:21]: can we add a e2e test? 
The expected use case is in deploy/v2/deploy_minterv2_follow.js  
e2e would look like:
1. deploy sgeth 
2. add secondary minter contract / eoa 
3. revoke deployer minter rights 
4. add secondary owner 
5. revoke deployer admin rights 
6. check auth invariants are preserved. i.e ex owner and outsiders cannot interact with the contract

File: .gitignore [hunk 3]
```diff
@@ -1,5 +1,6 @@
 node_modules
 .openzeppelin
+data
```
Comment (chimera-defi) [inline .gitignore]: why gitignore data?

File: .vscode/settings.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
+  "solidity.compileUsingRemoteVersion": "v0.8.20+commit.a1b79de6"
```
Comment (chimera-defi) [inline .vscode/settings.json]: lets add `.vscode` to the gitignore too so we dont commit this?

**Review by chimera-defi — CHANGES_REQUESTED** [review]

File: hardhat.config.js (hunk 1 above)
Comment (devlancer412) [reply hardhat.config.js:111]: seems like my prettier config changed this. let me try to revert this

File: test/v2/core/sgETH.spec.js (hunk 2 above)
Comment (devlancer412) [reply test/v2/core/sgETH.spec.js:21]: what's the secondary minter contract?

Comment (chimera-defi) [reply test/v2/core/sgETH.spec.js:21]: this one 
https://github.com/chimera-defi/SharedDeposit/blob/main/contracts/v2/core/SharedDepositMinterV2.sol

**Comment by chimera-defi** [comment]
youll need to create a new branch off of main. 
merged w/ some other minor changes in https://github.com/chimera-defi/SharedDeposit/pull/11

File: test/v2/core/sgETH.spec.js (hunk 2 above)
Comment (devlancer412) [reply test/v2/core/sgETH.spec.js:21]: what's the meaning of `revoke deployer minter rights`?

**Pull request merged**